### ISSUE TITLE
Focus does not stay on the same staff when using Ctrl F

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#29410]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1.  Open a score with multiple staves such as an orchestral score.
2. Click on or navigate to a note in a lower staff such as Viola in an orchestral score.
3. Use Ctrl F to jump to a later measure.
4. Notice that the focus goes to the designated bar but is now on the top staff rather than the staff the command was initiated from.

This is a significant barrier to composers of large scores who are blind and report it takes a lot of arrowing and loss of work flow/composition idea when getting back to the correct staff.

Focus should remain on the staff it was initiated from.  Note this is also mentioned in #19622  as a feature request to specify the staff as well as bar you want to jump to.  But not holding the position seems more like a separate issue to me.

### Supporting files, videos and screenshots

N/A

### What is the latest version of MuseScore Studio where this issue is present?

4.5

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by RMRattray** [comment]
Hello; I'll be taking this on shortly if I may.
